metrics.md

```diff
@@ -293,8 +293,69 @@ $$ O_i = \begin{cases}
+
+The monetary value of an improved forecast ($$ \text{value}_f $$) is then defined as:
+
+$$ \text{value}_f = \text{cost}_{f} - \text{cost}_{\text{ref}}, $$
+
+where $$ \text{cost}_f $$ and $$ \text{cost}_{\text{ref}} $$ refer to the costs of the selected forecast and reference forecast, respectively. Note that the choice of the reference forecast is crucial and should be consistent with current operational practices.
+
+
+### Production Cost Modeling
+{: .anchor }
+An alternative approach (not implemented by the Solar Forecast Arbiter) is to perform simulations using a production cost model (PCM) and then compare differences in costs incurred when using different forecasts. In addition to providing a more direct evaluation of the forecasts, simulations can provide insight into future value, e.g., how improved forecasts can improve system operations as solar penetration increases. However, such simulations require additional data dependencies and expertise that may not be readily available to forecasters.
+
+In order to the simulate the system, a PCM should be used to simulate the operations with and without energy storage. A number of key considerations for such simulations include:
+
+- **Use of multi-cycle models:** A multi-cycle model captures operations in at least two decision stages, such as day-ahead and real-time processes, and links the data together. For example a day-ahead decision may be made based on day-ahead forecasts and certain generators committed to provide the forecasted energy needs, plus any reserves. Then, the model updates to real time actuals and the system is redispatched, recognizing limitations on the ability to commit additional generation in response to errors. If such a model is used, the ability of an improved forecast to reduce startup of quick start units or reduce solar curtailment can be captured. An example of such a model is included in [Ela13](#ref-ela13) and [Martinez-Anido16](#ref-martinez-anido16).
+- **Use of dynamic reserves that reflect forecast errors:** As solar penetration increases, it is likely to impact reserves associated with balancing, such as regulation or ramping reserves. A number of ISOs and utilities are moving towards dynamically setting those reserves based on analysis of historical forecast error. Therefore, reducing forecast errors can result in reduced reserve requirements, which should also be included in simulations.
+
+Models that include the above can be used to assess value of forecasts, and have been exercised in previous studies, such as by NREL and others ([Zhang15](#ref-zhang15), [Wang16a](#ref-wang16a), [Wang16b](#ref-wang16b), [Wang17](#ref-wang17)). Such an approach provides a more extensive estimate of the value of improved forecasts. At the same time, they are still limited by simplifications made in any model, and are best used for directional analysis (i.e. specific $/MWh numbers are directional estimates and should be treated as such).
```

Comment:
> are best used for directional analysis (i.e. specific $/MWh numbers are directional estimates and should be treated as such).

I'm not quite sure what this means. That a specific $/MWh number is really only telling us the sign of the cost function?

Reply:
This was wording provided by Aidan (taken originally from the document he shared). My understanding is that `directional estimates` is in reference to these costs only telling you if you are reducing or increasing costs and should not be assumed as the "actual" final costs (since there are still many assumptions, even with a PCM). But I'll follow-up with Aidan to confirm and re-word to something that is clearer.

Reply:
I received clarification from Aidan: the `directional estimates` and related phrasing was meant to convey that the PCM-based analysis is good for order of magnitude or relative studies, but there are limits to the precision of the results (just like any model). For example, the PCM might say that reducing MAPE by 10% might reduce costs in a given system by 1.5%, but you can't say (with 100% certainty) whether it's exactly 1.5% or actually 3%; just that it's not likely negative and not 15%.

I'll revise the wording now.

Reply:
@wholmgren I revised the wording (in an attempt) to be clearer on how to interpret PCM results.

Reply:
much more clear now.